syntax = "proto3";
package message;
message GetVarifyReq {
  string email = 1;
}
message GetVarifyRsp {
  int32 error = 1;
  string email = 2;
  string code = 3;
}
message GetChatServerReq {
  int32 uid = 1;
}
message GetChatServerRsp {
  int32 error = 1;
  string host = 2;
  string port = 3;
  string token = 4;
}
service VarifyService {
  rpc GetVarifyCode (GetVarifyReq) returns (GetVarifyRsp) {}
}
service StatusService {
    rpc GetChatServer (GetChatServerReq) returns (GetChatServerRsp) {}
}